examples/js/src/main/scala/doodle/examples/canvas/ConcentricCircles.scala

```diff
@@ -20,21 +20,20 @@ import cats.effect.unsafe.implicits.global
 import doodle.canvas.{*, given}
 import doodle.core.*
 import doodle.syntax.all.*
+import doodle.image.*
 
 import scala.scalajs.js.annotation.*
 
 @JSExportTopLevel("CanvasConcentricCircles")
 object ConcentricCircles {
-  def circles(count: Int): Picture[Unit] =
-    if count == 0 then
-      Picture.circle(20).fillColor(Color.hsl(0.degrees, 0.7, 0.6))
-    else
-      Picture
-        .circle(count.toDouble * 20.0)
-        .fillColor(Color.hsl((count * 15).degrees, 0.7, 0.6))
-        .under(circles(count - 1))
+  
+  def rect1 = Image.rectangle(100, 100).fillColor(Color.red)
```

Comment:
You've broken the example. Please undo this. If you want to test your own code you can easily create another example.